apps/super-app/src/routes/SupportDetailPage.tsx

```diff
@@ -76,17 +95,18 @@ function Blurb(props: BlurbProps) {
 }
 
 type DescriptionProps = Pick<
-  Twc2SupportDetail,
+  SupportDetailWithLinks,
   'logo' | 'name' | 'website' | 'blurbIntro' | 'blurbFacebookLinks'
 >;
 
 function DescriptionSection(props: DescriptionProps) {
   const { logo, name, website, blurbIntro, blurbFacebookLinks } = props;
   return (
+    // TODO: support logo is not properly constrained in some cases
```

Comment:
What do you mean by this, and how will we know when we've fixed it?

Reply:
@glencbz I discovered it when I tried adding the IRR logo into the Details Page. It blew up instead of being constrained to the max-height. Likely coz the logo was tall aspect ratio wise.

We can simply square all our logos to circumvent this actually.

Reply:
That works, though I think there is probably an elegant CSS-based solution for this (e.g. setting the `height` of the containing element). We don't have to discuss this now; we can defer this work until we need it.